test/bin/test-citgm-all.js

```diff
@@ -3,9 +3,9 @@
 var path = require('path');
 
 var test = require('tap').test;
-var spawn = require('../lib/spawn');
+var spawn = require('../../lib/spawn');
 
-var citgmAllPath = path.resolve(__dirname, '..', 'bin', 'citgm-all');
+var citgmAllPath = path.resolve(__dirname, '..', '..', 'bin', 'citgm-all.js');
```

Comment:
ohhh, I was unaware of require.resolve. I like that I don't need to be all particular about the path to be safe for windows.